docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -276,6 +276,26 @@ By default, access from `localhost` and `127.0.0.1` is accepted.
 !!!tip
     To allow all hostnames, use `"*"`. We don't recommend allowing all hostnames for production code.
 
+### identity
+
+```bash tab="Syntax"
+--identity=<String>
+```
+
+```bash tab="Command Line"
+--identity=MyNode
+```
+
+```bash tab="Environment Variable"
+BESU_IDENTITY=MyNode
+```
+
+```bash tab="Configuration File"
+identity="MyNode"
+```
+
+Name for the node. If specified, it is the second part of the client ID provided by some Ethereum network explorers. For example, in the client ID `besu/MyNode/v1.3.4/linux-x86_64/oracle_openjdk-java-11`, the node name is `MyNode`.
+
```

Comment:
I think we need some verbiage to set expectations if it is not set.
```suggestion
If this value is not set then the client ID will not have a node name section, for example `besu/v1.3.4/linux-x86_64/oracle_openjdk-java-11`.
```

Reply:
I revised your suggestion to conform with some of our doc standards.